swc/private/swc.bzl

```diff
@@ -291,14 +289,11 @@ def _swc_impl(ctx):
                 js_map_out = ctx.actions.declare_file(map_out_path)
                 outputs.append(js_map_out)
 
-            inputs = [src]
-            inputs.extend(ctx.toolchains["@aspect_rules_swc//swc:toolchain_type"].swcinfo.tool_files)
-            inputs.extend(ctx.files.plugins)
-            inputs.extend(plugin_cache)
+            src_inputs = [src] + inputs
```

Comment:
how come we no longer need to include the `swcinfo.tool_files`?

Reply:
Everything is already in `inputs`. See the first couple lines of `_swc_impl`